src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -56,8 +56,7 @@
 _COMMON_TENANT = 'common'
 
 _AUTH_CTX_FACTORY = lambda authority, cache: adal.AuthenticationContext(authority,
-                                                                        cache=cache,
-                                                                        api_version=None)
+                                                                        cache=cache)
```

Comment:
@brendandburns Run `python scripts/dev_setup.py` in your environment so that you have the latest dependencies as our adal version was recently updated and that's why you see an error.